chi/network.py

```diff
@@ -663,13 +671,14 @@ def get_floating_ip(ip_address) -> dict:
     Returns:
         The floating IP representation.
     """
-    fip = next(iter([
-        fip for fip in list_floating_ips()
-        if fip['floating_ip_address'] == ip_address
-    ]), None)
-    if not fip:
-        raise ValueError(f'No floating IP with address {ip_address} found')
-    return fip
+    ips = neutron().list_floatingips()['floatingips']
+    
+    for fip in ips:
+        if fip['floating_ip_address'] == ip_address:
+            return fip
+    print("Floating ip not found " + ip_address)
```

Comment:
Same comment here, I prefer to `raise`; one can't assume that `print` works as you expect. There should be no `print` calls anywhere in python-chi as a rule, in fact (there may be now, but there should not be.) Printing to the console is something the caller can do if it wants.

Reply:
Incorporated raise Exception(str) in place of print.